packages/client/src/SharedClient.ts

```diff
@@ -8,11 +8,14 @@ import { DirectoryClient } from "./DirectoryClient.js";
 import { Endpoint, Token } from "./Http.js";
 import { NetworkState } from "./NetworkState.js";
 import { LegalOfficerClass } from "./Types.js";
+import { UUID } from "@logion/node-api/dist/types/UUID";
```

Comment:
```suggestion
import { UUID } from "@logion/node-api";
```

The place where `UUID` is defined is an implementation detail.

Reply:
Agreed. (F*****g IDE auto-completion !)